xdg-ref-engine-discovery.md

```diff
@@ -0,0 +1,72 @@
+# OCI XDG Ref-Engine Discovery
+
+This is version 0.1 of this specification.
+
+To faciliate local control over image resolution, this specification defines a [ref-engine discovery](glossary.md#ref-engine-discovery) protocol which publishers MAY use to direct consumers towards [reference engines](glossary.md#ref-engine).
```

Comment:
This needs rewording to drop "publishers".  The XDG spec is just about local configuration.

Reply:
> This needs rewording to drop "publishers".

Done with b971df3 → 72f9f49.

Reply:
> > This needs rewording to drop "publishers".
>
> Done with b971df3 → 72f9f49.

Oops, I'd forgotten to push the last fixup.  This was actually done with 72f9f49 → 2a297c5.